pkg/node/node.go

```diff
@@ -1050,10 +1072,26 @@ func (s *Service) NodeGetInfo(ctx context.Context, req *csi.NodeGetInfoRequest)
 							} else {
 								nvmetcpConnectCount = nvmetcpConnectCount + 1
 							}
+
+							svcID, err := strconv.ParseInt(target.TrsvcID, 10, 64)
```

Comment:
why we connecting both using TCP and DPU ? DPU is instead of TCP, if DPU is present we should use it instead... like with FC and TCP... you use either one, not both...